addons/website_sale/tests/test_address.py

```diff
@@ -660,6 +660,38 @@ def test_12_recompute_taxes_on_address_change(self):
             "Tax should no longer change after order confirmation",
         )
 
+    def test_two_types_address_updates(self):
+        user = self.user_portal
+        user_partner = user.partner_id
+
+        so = self._create_so(partner_id=user_partner.id)
+
+        invoicing = self.env['res.partner'].create([
+            {
+                'name': 'Invoicing',
```

Comment:
Same thing regarding invoicing
```suggestion
                'name': 'Test Address',
```